elifetools/file_utils/__init__.py

```diff
@@ -19,10 +20,22 @@ def sample_xml(filename):
 def fixture_folder(folder_name):
     return os.path.join(BASE_DIR, "tests", "fixtures", folder_name)
 
+def fixture_module_name(folder_name, filename):
+    return '.'.join(["tests", "fixtures", folder_name, filename.rstrip('.py')])
+
+def fixture_file(folder_name, filename):
+    return os.path.join(fixture_folder(folder_name), filename)
+
 def read_fixture(folder_name, filename):
-    full_filename = os.path.join(fixture_folder(folder_name), filename)
-    with open(full_filename) as file_fp:
-        return unicode_value(file_fp.read())
+    full_filename = fixture_file(folder_name, filename)
+    if full_filename.endswith('.py'):
+        # import the fixture and return the value of expected
+        module_name = fixture_module_name(folder_name, filename)
+        mod = importlib.import_module(module_name)
```

Comment:
could check that `mod.expected` exists (`getattr`?) to make it foolproof

Reply:
Good idea, I'm committing with an assertion the property exists in the module before continuing.